icaruscode/Generators/ldm_g4_icarus.fcl

```diff
@@ -0,0 +1,86 @@
+#include "services_icarus_simulation.fcl"
```

Comment:
What is this configuration adding/changing to the functionality of the standard G4 (`standard_g4_icarus.fcl`)?
If it is only the `mcreco` module, and that one is intentionally desired, I _recommend_ that this configuration inherits the defaults from the experiment and just add it:
```
#include "mcreco.fcl"
#include "standard_g4_icarus.fcl"

physics.producers.mcreco: @local::standard_mcreco  # from mcreco.fcl
physics.producers.simulate: [ @local::physics.producers.simulate, mcreco ]
```